agents/a2a_mcp_integration.py

```
@@ -427,6 +425,71 @@
+                        logger.error(
+                            f"MCP tool call failed: {
+                                response.status}"
+                        )
+                        return {
+                            "status": "error",
+                            "error": f"HTTP {response.status}",
+                            "tool": tool_name,
+                        }
+
+        except aiohttp.ClientError as e:
+            logger.error(f"MCP connection error: {e}")
```

Ruff (0.11.9)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)